tests/ztest/ztress/testcase.yaml

```diff
@@ -0,0 +1,5 @@
+common:
+  tags: test_framework
+tests:
+  testing.ztest.ztress:
+    platform_allow: qemu_x86
```

Comment:
It would be great if this not only run on qemu_x86.  Testing on multiple CPUs might not be easy but looking forward to seeing it in the future.
At this moment we can use CONFIG_MP_NUM_CPUS = 1 and CONFIG_TEST_RANDOM_GENERATOR=y to make it run for all boards. Here is what I have tried:
```
ERROR   - qemu_nios2                tests/ztest/ztress/testing.ztest.ztress            FAILED: Timeout
ERROR   - see: /home/mm/zephyrproject/zephyr/twister-out/qemu_nios2/tests/ztest/ztress/testing.ztest.ztress/handler.log
INFO    - Total complete:   21/  23  91%  skipped:   14, failed:    1
ERROR   - qemu_cortex_a53           tests/ztest/ztress/testing.ztest.ztress            FAILED: Timeout
ERROR   - see: /home/mm/zephyrproject/zephyr/twister-out/qemu_cortex_a53/tests/ztest/ztress/testing.ztest.ztress/handler.log
INFO    - Total complete:   22/  23  95%  skipped:   14, failed:    2
ERROR   - qemu_cortex_a53_smp       tests/ztest/ztress/testing.ztest.ztress            FAILED: Timeout
ERROR   - see: /home/mm/zephyrproject/zephyr/twister-out/qemu_cortex_a53_smp/tests/ztest/ztress/testing.ztest.ztress/handler.log
INFO    - Total complete:   23/  23  100%  skipped:   14, failed:    3
```
That's my personal suggestion, thanks.

Reply:
Thanks for pointing that out. I've updated test to be run on all platforms with qemu_x86 as intergration platform. The reason why i used qemu_x86 (and it will be recommended to use for stress testing) is that you can increase system clock frequency and it executes quite fast. And you need higher frequency to increase chance of preemptions.